static/app/views/insights/common/utils/centerTruncate.ts

```diff
@@ -1,16 +0,0 @@
-import {formatVersion} from 'sentry/utils/versions/formatVersion';
-
-export const ELLIPSIS = '\u2026';
-
-function centerTruncate(value: string, maxLength = 20) {
-  const divider = Math.floor(maxLength / 2);
-  if (value?.length > maxLength) {
-    return `${value.slice(0, divider)}${ELLIPSIS}${value.slice(value.length - divider)}`;
-  }
-  return value;
-}
-
-export function formatVersionAndCenterTruncate(value: string, maxLength?: number) {
-  const formattedVersion = formatVersion(value);
-  return centerTruncate(formattedVersion, maxLength);
-}
```

Comment:
file renamed to `static/app/views/insights/common/utils/formatVersionAndCenterTruncate.ts`